src/plugins/recordTypes/intake/forms/default.jsx

```diff
@@ -0,0 +1,155 @@
+const template = (configContext) => {
+  const {
+    React,
+  } = configContext.lib;
+
+  const {
+    Col,
+    Cols,
+    Panel,
+    Row,
+  } = configContext.layoutComponents;
+
+  const {
+    Field,
+    InputTable,
+  } = configContext.recordComponents;
+
+  return (
+    <Field name="document">
+      <Panel name="objectEntryInfo" collapsible>
+        <Cols>
+          <Col>
+            <Row>
+              <Field name="entryNumber" />
+              <Field name="entryDate" />
+            </Row>
+            <Field name="expectedReturnDate" subpath="ns2:intakes_pahma" />
+
+            <InputTable name="depositor">
+              <Field name="depositor" />
+              <Field name="depositorsRequirements" />
+            </InputTable>
+
+            <Field name="currentOwner" />
+          </Col>
+          <Col>
+            <Row>
+              <Field name="entryMethods">
+                <Field name="entryMethod" />
+              </Field>
+
+              <Field name="entryReason" />
+            </Row>
+
+            <Field name="returnDate" />
+            <Field name="entryNote" />
+          </Col>
+        </Cols>
+
+        <Field name="packingNote" />
+      </Panel>
+
+      <Panel name="objectCollectionInfo" collapsible collapsed>
+        <Cols>
+          <Col>
+            <Row>
+              <Field name="fieldCollectionDate" />
+
+              <Field name="fieldCollectionMethods">
```

Comment:
Need to wrap fieldCollectionMethods in a `Col`. Notice there's no margin between the fields in the row. Wrapping the repeating field in a Col works around this.